NOTES.md

```diff
@@ -1,3 +1,16 @@
+# Release 1.16.2 (Stand By Me)
+
+Released on 2015-02-16
+
+Bug fix update.
+
+* Fixed a bug in `r.range` that caused a crash in debug mode (#3767)
```

Comment:
This would also cause queries to fail in release mode - so I would just say "that caused query failures"